website/docs/docs/dbt-versions/2024-release-notes.md

```diff
@@ -9,6 +9,10 @@ pagination_prev: null
 
 dbt Cloud release notes for 2024.
 
+## May 2024
+
+- **Behavior Change:** Introduced [`require_resource_names_without_spaces`](/docs/reference/global-configs/legacy-behaviors#require_resource_names_without_spaces), opt-in and disabled by default. If set to `True`, dbt will raise an exception if it finds a resource name containing a space in your project or an installed package. This will become the default in a future version of dbt.
```

Comment:
@mirnawong1 Since these are new, and I'm adding manually, I'm not sure if this is the right way to do it — feel free to edit/update directly!

Reply:
no worries @jtcohen6 !